src/app/onboarding/components/create-new-archive/create-new-archive.component.ts

```diff
@@ -244,11 +251,22 @@ export class CreateNewArchiveComponent implements OnInit, OnDestroy {
         },
       },
     });
-    this.dialog.open(
-      'SkipOnboardingDialogComponent',
-      { skipOnboarding: this.skipOnboarding },
-      { width: '600px' }
-    );
+    if (!this.isGlam) {
+      this.dialog.open(
+        'SkipOnboardingDialogComponent',
+        { skipOnboarding: this.skipOnboarding },
+        { width: '600px' }
+      );
+    } else {
+      this.screen = 'create-archive-for-me';
+    }
+  }
+
+  public navToGoals(event: Record<string, string>): void {
```

Comment:
I was hoping that the `Record<string, string>` type could be eliminated throughout this whole module. Could you make the ArchiveCreateEvent type exportable so that this typing can be used throughout this code?